xml/System.Net.Http/WinHttpHandler.xml

```diff
@@ -316,15 +316,14 @@ When this property is set to `true`, all HTTP redirect responses from the server
         <ReturnType>System.Net.ICredentials</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets the credentials used to authenticate the user to an authenticating proxy.</summary>
-        <value>To be added.</value>
+        <summary>When the default (system) proxy is used, gets or sets the credentials used to submit to the default proxy server for authentication.</summary>
+        <value>The credentials used to authenticate the user to an authenticating proxy.</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- The default value for this property is <xref:System.Net.CredentialCache.DefaultCredentials?displayProperty=nameWithType>.  
-  
- ]]></format>
+          <format type="text/markdown"><![CDATA[
+
+The default proxy is used only when <xref:System.Net.Http.WinHttpHandler.Proxy> is set to `null`.
```

Comment:
```suggestion
The default proxy is used only when <xref:System.Net.Http.WinHttpHandler.WindowsProxyUsePolicy> is set to   <xref:System.Net.Http.WindowsProxyUsePolicy.UseWinHttpProxy> and  <xref:System.Net.Http.WinHttpHandler.Proxy> is set to `null` (both are the default values of the respective properties).
```

Reply:
Technically, the default proxy is used only if `WindowsProxyUsePolicy`=`UseWinHttpProxy`; and the validation won't allow you to set `Proxy` in that case...

But if you wanted to specifically highlight the null check we've seen, maybe we could then say it from a different angle:

```suggestion
The value of that property is used only when <xref:System.Net.Http.WinHttpHandler.Proxy> is set to `null`.
```

Reply:
I removed the remarks completely, it's not that straight forward as just setting `UseWinHttpProxy`, even `UseCustomProxy` can (under certain conditions) lead to using `DefaultProxyCredentials`.